scaling_performance/cns_guidance.adoc

```diff
@@ -0,0 +1,82 @@
+[[scaling-performance-cns-guidance-for-databases]]
+= CNS Guidance For Databases
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[CNS-Guidance-For-Databases]]
+== CNS Guidance for Databases
+In this document will be listed best practices and guidance we recommend 
```

Comment:
When you use container native storage (CNS) for applications, follow the guidance and best practices provided in this topic.